main.go

```diff
@@ -34,7 +34,8 @@ func init() {
 
 var (
 	listenAddress       = flag.String("web.listen-address", ":9102", "The address on which to expose the web interface and generated Prometheus metrics.")
-	metricsEndpoint     = flag.String("web.telemetry-path", "/metrics", "Path under which to expose metrics.")
+	metricsEndpoint     = flag.String("web.telemetry-path", "/statsd", "Path under which to expose metrics.")
+	intMetricsEndpoint   = flag.String("web.int-telemetry-path", "/metrics", "Path under which to expose metrics.")
```

Comment:
Please update the help string for both flags to make the difference very very clear.